core/templates/dev/head/exploration_editor/main_tab/StateInteraction.js

```diff
@@ -64,6 +66,11 @@ oppia.controller('StateInteraction', [
     $scope.stateInteractionIdService = stateInteractionIdService;
     $scope.hasLoaded = false;
 
+    $scope.userBlueImgUrl = UrlInterpolationService.getStaticImageUrl(
+                              '/avatar/user_blue_72px.png');
```

Comment:
Ditto re indentation, here and elsewhere.

Do you think it might be better to have UrlInterpolationService.getDefaultUserAvatarImageUrl()? We seem to be asking for this avatar a lot in many places. Not sure though.